src/encoded/static/components/filegallery.js

```diff
@@ -670,7 +673,15 @@ class RawSequencingTable extends React.Component {
                                         // Must be an index reads file if it's not paired-ended.
                                         pairClass = 'index-reads';
                                     }
-                                    if (groupFiles[i + 1] && groupFiles[i + 1].pairSortId !== file.pairSortId) {
+
+                                    // Draw bottom border if second file of pair and no index reads
+                                    // follow, or file *is* index reads, but not if last row of
+                                    // table.
+                                    const isNextInGroupIndexReads = groupFiles[i + 1] && groupFiles[i + 1].output_type === 'index reads';
+                                    if (
```

Comment:
Would it be a good idea to place each condition in the if-statement in a variable and use the if-statement on those variables? That has the advantage of keeping the readers' attention focus and may help reduce the length. That is something I picked up from Python.